code/modules/vehicles/cm_armored.dm

```diff
@@ -439,7 +439,7 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 	if(.)
 		for(var/mob/living/M in get_turf(A))
 			//I don't call Bump() otherwise that would encourage trampling for infinite unpunishable damage
-			M.sleeping = 1e7 //Maintain their lying-down-ness
+			M.sleeping = 5 //Maintain their lying-down-ness
```

Comment:
I hate how all of this works with the multi-tile tanks T.T

Reply:
We both do. It's just a temporary patch because this boat is full of holes and there's water entering everywhere.